src/stylus/fablevision/clutch-game.styl

```diff
@@ -1,18 +1,139 @@
+
+  .geniblocks.parentDrake
+    height: 240px
+    margin: -17px 0 -45px 0
+    position: relative
+    &.mother, &.father
+      margin-right: 0px
+      margin-left: 0px
+      .parentTitle
+        position: absolute
+        top:  56px
+        color: #f9ec9a
+      .organism
+        position: absolute
+        top: -22px
+
+    &.mother
+      background-image: none
+      left: 53px
+      .parentTitle
+        left: -10px
+      .organism
+        left: 112px
+
+    &.father
+      background-image: none
+      left: -33px
+      .parentTitle
+        left: 346px
+      .organism
+        left:  64px
+
+  .geniblocks.genome.parent
+    margin-top: 40px
 
   .geniblocks.target-drake-container
-    background-image: url('../resources/fablevision/layout-b/target_gizmo_single.png')
-    margin-top: -853px
+    background-image: url('../resources/fablevision/layout-h/target_gizmo_single.png')
+    margin: -87px 0 17px 0
+    // margin-top: -87px;
     width: 400px
     height: 362px
-    background-size: contain
-    background-repeat: no-repeat
     flex-direction: column
+    align-items: center
     .organism
-      margin-top: 62px
-      margin-left: 100px
+      margin: 65px 0 0 -14px
     .target-drake-text
-      margin-top: -25px
-      margin-left: 143px
+      margin-top: -4px
+      margin-left: 5px
+
+  .geniblocks.clutch-view
+    margin-top: -3px
+    height: 323px
+    width: 577px
+    background: none
+
+    .clutch-slider
+      width: 169px
+      height: 29px
+      background-repeat: no-repeat
+      background-position-y: 50%
+      &.left
+        background-image: url('../resources/fablevision/layout-b/Clutch_Slider_Arrow_L.png'),
+                          url('../resources/fablevision/layout-h/slider_track.jpg')
+      &.right
+        background-image: url('../resources/fablevision/layout-b/Clutch_Slider_Arrow_R.png'),
+                          url('../resources/fablevision/layout-h/slider_track.jpg')
+        background-position-x: 100%
+
+    .clutch-counter
+      width: 100%
+      justify-content: space-between
+      margin-top: 5px
+      z-index: 10       // there's a slight overlap with lower drakes
+
+      .stable-count
+        margin-left: 90px
+
+    .clutch-slider
+      margin-top: 0
+
+    .clutch-page-count
+        margin-left: 0px
+
+    .drake-pages
+      height: 80%
+
+      .clutch-drakes
+        display: flex
+        flex-direction: row
+        height: 100%
+        margin: 0
+        align-self: flex-end
+        justify-content: space-around
+        align-content: space-around
+
+        .stable-drake-overlay
+          background-image: url('../resources/fablevision/layout-h/pen_circle.png')
+          margin: 0
+
+  .geniblocks.breed-button-area
+    background-image: none
+    height: auto
+    width: auto
+    margin-top: 23px
+
+  .geniblocks.breed-button-surround
+    background: none
+    height: auto
+    width: auto
+    margin-top: 0
+
+    .breed-button
```

Comment:
There's a slight issue with the breed button when you click and hold - the image switches but the image appears tiled and off-center. Perhaps this is related to the different image sizes?

Reply:
:+1: I had noticed this with the other button sizes, and got new versions from FV, but forgot to update the breed button.

This should be fixed now (and visible when Travis has built it).

Reply:
I think we have a case-sensitivity issue - Breed_ON vs breed_ON (and similarly for the other variants) - the built version of the code responds with a 404 for the mismatch